### PR TITLE
Clarify proof mechanism and algorithm selection

### PR DESCRIPTION
**Pull request by ottonomy**
Phil alerted me that we may have left some things about securing mechanisms and how they may evolve over time unclear. I've taken a stab at clarifying what is required for conformance, and how experimental forward-looking usage of additional cryptosuites may evolve and eventually be recommended by the workgroup for inclusion in the conformance suite and core spec itself, given that 1EdTech confirmed prior to finalization that it may be possible to allow additional cryptosuites as an errata-level change (not rising to the level of a new chartered minor spec version).

* [x] Disambiguate 1EdTech conformance certification from experimental or coordinated usage
* [x] Update implementation guide
* [x] Spelling corrections in affected page sections.
* [x] TODO: update errata document

### CODE REVIEW

**Comment by andyfmiller** [comment]
Great improvements Nate!

It's been a while since I participated in the discussions, and was a little confused by the section on [Achievement.id DataURIs](https://github.com/1EdTech/openbadges-specification/diffs/0?base_sha=bce0cd4cfef6de1c46769ea2abff3fa9a7ee23e0&branch=feature%2Fclarify-data-integrity-options&commentable=true&name=feature%2Fclarify-data-integrity-options&pull_number=584&qualified_name=refs%2Fheads%2Ffeature%2Fclarify-data-integrity-options&sha1=bce0cd4cfef6de1c46769ea2abff3fa9a7ee23e0&sha2=cc1cc79c8a0f6446f3ee37230d355c5351597257&short_path=5868c36&unchanged=expanded&w=false#embedding-evidences).

Maybe the Implementation Guide could be updated to explain why one would use a DataURI. [Section 3.1.3](https://www.imsglobal.org/spec/ob/v3p0/impl/#publishing-achievement-definitions-and-selecting-achievement-identifiers) does not mention it.

**Comment by ottonomy** [comment]
@andyfmiller about the data URIs mention in the diff, sorry that's unrelated to this PR. I did correct a few spelling errors that I found while doing this work, and my editor rewrapped some lines on save, and I missed reverting that one. I'll take another look and make sure to remove extraneous changes in a second commit.

**Comment by longpd** [comment]
The primary concern here is that the to use a verifiable credential linked data proof or data integrity method the current spec says you [MUST use the EdDCA](https://www.imsglobal.org/spec/ob/v3p0/#lds-proof:~:text=validUntil%2C%20respectively-,8.3,-Linked%20Data%20Proof) cryptographic proof. However, that proof method while relatively simple, also exposes the credential to potential identity discovery through linking the paths that a credential follows (issuer to credentialSubject; holder to relying party, e.g., employer; and relying party to verifier).

It's a core design tenet of the VC ecosystem to try where possible to build unlinkable credentials so that discovering the identity of a credentalSubject, or an issuer, or a relying party is simply not possible. There are crypto suites the offer this. For VCDM v2.0, the one most robust in its capabilities is [BBS](https://www.w3.org/TR/vc-di-bbs/), now a part of the recommended data integrity options for VCs.

For some use cases unlinkability may not be important. But for many it is. [Two examples from the BBS spec illustrate these two circumstances](https://www.w3.org/TR/vc-di-bbs/#selective-disclosure-and-unlinkability:~:text=Selective%20Disclosure%20and-,Unlinkability,-In%20some%20uses). 

I understand that the it quickly becomes awkward to build conformance tests to be able to say that a given issuer's implementation does meet the 1Edtech standard. We want to support commemorating compliance. But we also want to support use cases where discoverable identity needs to be prevented.  

One option is to do what the Federal Government has done. The Feds require that official business using credentials must demonstrate compliance with their security guidelines, even though those guidelines do not represent the current state of the cryptographic arts. 

The work around, sanctioned by the US Gov, is to demonstrate compliance with a signature method within the suite of those required, but allow use of another signature method in practice. This 'dual signature' approach allows them to say the credential adhered to the mandated requirements, but the alternate proof can be used in real world applications. 

In our case, that is, for 1Edtech's case, one could have a conformance test that uses EdDSA in order to demonstration the required guidelines have been met and enable the issuance of a 1Edtech endorsement of compliance. We could also indicate that there are a suite of tested VCDM v2.0 compliant data integrity suites for linked data proofs that implementers should be able to select among for their use cases. If unlinkability is among them, then BBS is appropriate for them. Whether we formally require that the credentials are dual signed our not is entirely up to 1Edtech. I'd suggest that dual signature is not needed in actual implementation. 

The recommendation here would be we make the EdDSA necessary for conformance testing, but we allow any of the designated data integrity suites that are compliant with the VCDM v2.0 specification. The Implementation Guide can then elaborate more about those options.

**Comment by justinpitcher** [comment]
+1 approve this PR.

File: ob_v3p0/integrity.md [hunk 1]
```diff
@@ -161,12 +161,12 @@ This standard supports the Linked Data Proof format. In order to opt for this fo
 Perform these steps to attach a Linked Data Proof to the credential:
 
 1. Create an instance of [Multikey](#multikey) as shown in [Section 2.1.1 DataIntegrityProof](https://www.w3.org/TR/vc-di-eddsa/#multikey) of [[VC-DI-EDDSA]].
-1. Using the key material, sign the credential object as shown in [Section 7.1 Proof Algothim](https://w3c-ccg.github.io/data-integrity-spec/#proof-algorithm) of [[DATA-INTEGRITY-SPEC]] to produce a [Proof](#proof) as shown in [Section 2.2.1 DataIntegrityProof](https://www.w3.org/TR/vc-di-eddsa/#dataintegrityproof) of [[VC-DI-EDDSA]] with a \`proofPurpose\` of "assertionMethod".
+1. Using the key material, sign the credential object as shown in [Section 7.1 Proof Algorithm](https://w3c-ccg.github.io/data-integrity-spec/#proof-algorithm) of [[DATA-INTEGRITY-SPEC]] to produce a [Proof](#proof) as shown in [Section 2.2.1 DataIntegrityProof](https://www.w3.org/TR/vc-di-eddsa/#dataintegrityproof) of [[VC-DI-EDDSA]] with a \`proofPurpose\` of "assertionMethod".
```
Comment (xaviaracil) [inline ob_v3p0/integrity.md:164]: This paragraph is constrained to VC-DI-EDDSA, which is excellent nowadays but could be wrong in the future. Maybe replacing [[VC-DI-EDDSA]] with `one of the supported suites listed in the Conformance and Certification Guide`

**Pull request merged**